src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala

```diff
@@ -30,21 +30,11 @@ class ImapFullScenario {
     // the corresponding issue can be found at :  https://github.com/linagora/gatling-imap/issues/38
     .exec(imap("fetch").fetch(MessageRanges(Last()), AttributeList("UID", "BODYSTRUCTURE")).check(ok))
 
-  val heavyUser = repeat(3)(receiveEmail)
-    .repeat(2)(readLastEmail)
-  val lightUser = receiveEmail
-    .exec(readLastEmail)
-
   def generate(duration: Duration, feeder: FeederBuilder): ScenarioBuilder =
     scenario("Imap")
       .feed(feeder)
-      .pause(1.second)
       .exec(imap("Connect").connect()).exitHereIfFailed
       .exec(imap("login").login("${username}", "${password}").check(ok))
-      .during(duration) {
-        randomSwitch(
-          75.0 -> exec(lightUser),
-          25.0 -> exec(heavyUser)
-        )
-      }
-}
\ No newline at end of file
+      .exec(receiveEmail
+        .exec(readLastEmail))
```

Comment:
you don't loop at all?

Reply:
why not
```
.exec(receiveEmail)
.exec(readLastEmail)
```
 ?

Reply:
why did you remove different usages (light / heavy)?

Reply:
I think the best way would be to inject two different profiles in the simulation the light users and the heavy ones.
The light one would play a ImapFullLightScenario and the heavy ones a ImapFullHeavyScenario.

What do you think of that ?

Reply:
I think he removed light and heavy users to drop the duration of the ImapFullScenario, as in the title... I'm not sure though to understand the point of that like the others?

Reply:
nice solution, this will better balance the scenarios

Reply:
> I think he removed light and heavy users to drop the duration of the ImapFullScenario, as in the title... I'm not sure though to understand the point of that?

Yes, we should drop the duration in the scenario. My proposal is, in addition to that to drop the switch in the scenario. Make 2 different scenario for each usage. And in the simulation inject 75% of users with the light scenario and 25% with the heavy one. 
I hope its clearer.

Reply:
Oh ok ! Thanks for the clarification @remk , still not familiar enough with the gatling simulations and scenarios :)